ui/app/components/clients/attribution.js

```diff
@@ -1,45 +1,105 @@
-    };
-    // if byAuthMethod
-    // return
-    // byAuthMethod = {
-    //   description: "This data shows the top ten authentication methods by client count within this namespace, and can be used to understand where new clients and total clients are originating. Authentication methods are organized by path.",
-    //   newCopy: `The new clients used by the auth method for this {{@range}}. This aids in understanding which auth methods create and use new clients ${this.dateRange === "date range" ? " over time." : "."}`,
-    //   totalCopy: `The total clients used by the auth method for this ${this.dateRange}. This number is useful for identifying overall usage volume. `
-    // }
+    let dateText = this.isDateRange ? 'date range' : 'month';
+    if (this.isAllNamespaces) {
+      return {
+        description:
+          'This data shows the top ten namespaces by client count and can be used to understand where clients are originating. Namespaces are identified by path. To see all namespaces, export this data.',
+        newCopy: `The new clients in the namespace for this ${dateText}. 
+          This aids in understanding which namespaces create and use new clients 
+          ${dateText === 'date range' ? ' over time.' : '.'}`,
+        totalCopy: `The total clients in the namespace for this ${dateText}. This number is useful for identifying overall usage volume.`,
+      };
+    } else if (!this.isAllNamespaces) {
+      return {
+        description:
+          'This data shows the top ten authentication methods by client count within this namespace, and can be used to understand where clients are originating. Authentication methods are organized by path.',
+        newCopy: `The new clients used by the auth method for this ${dateText}. This aids in understanding which auth methods create and use new clients 
+        ${dateText === 'date range' ? ' over time.' : '.'}`,
+        totalCopy: `The total clients used by the auth method for this ${dateText}. This number is useful for identifying overall usage volume. `,
+      };
+    } else {
+      return {
+        description: 'There is a problem gathering data',
+        newCopy: 'There is a problem gathering data',
+        totalCopy: 'There is a problem gathering data',
+      };
+    }
+  }
+
+  get getCsvData() {
+    let results = '',
+      data,
+      fields;
+    if (this.isDateRange) {
+      data = this.args.totalClientsData;
+      fields = [`${this.clientCountBreakdown}`, 'Active clients', 'Unique entities', 'Non-entity tokens'];
+    } else {
+      data = this.args.newClientsData;
+      // will need add a column for new clients
+      fields = [`${this.clientCountBreakdown}`, 'Active clients', 'Unique entities', 'Non-entity tokens'];
+    }
+
+    results = fields.join(',') + '\n';
+    data.forEach(function (item) {
+      // debugger
```

Comment:
just left over? maybe remove unless you need it later

Reply:
I can remove! I left behind since this function will change quite a bit once data format is finalized